t/runtime-bareword-filehandles.t

```diff
@@ -27,4 +27,6 @@ ok(
     'Successfully handled bareword filehandles during runtime',
 );
 
+is( "$@", '', 'No observed error' );
```

Comment:
does lives clear $@?

Reply:
Yes. At first, I tried to turn the test into using `lives()`, but that didn't work. When I read up on it, it says that it leaves `$@` untouched so you could test it.

From the docs:

> $@ is preserved from before the sub is called when there is no exception. When an exception is trapped $@ will have the exception so that you can look at it.